src/Sidekick.Apis.Poe/Parser/Properties/PropertyParser.cs

```diff
@@ -39,10 +39,13 @@ IStringLocalizer<FilterResources> filterLocalizer
 
     private Regex? AreaLevel { get; set; }
 
+    private double NormalizeValue { get; set; }
+
     public async Task Initialize()
     {
         var leagueId = await settingsService.GetString(SettingKeys.LeagueId);
         var game = leagueId.GetGameFromLeagueId();
+        NormalizeValue = await settingsService.GetObject<double>(SettingKeys.PriceCheckNormalizeValue);
```

Comment:
Initialize() is called only once during startup, we want to be able to change the settings and have the updated setting for subsequent searches. Keep this in the GetFilters() method.